src/packages/mikroorm/src/database.ts

```diff
@@ -39,7 +39,7 @@ const NumericIsolationLevels = {
 	[IsolationLevel.READ_UNCOMMITTED]: 1,
 };
 
-class DatabaseImplementation {
+export class DatabaseImplementation {
```

Comment:
Not sure I'm keen on exposing this. Is it just for the type? if so, we probably want to export an interface I reckon.

Reply:
ok cool ill just export the type as thats all I need